src/styles/book.css

```diff
@@ -0,0 +1,125 @@
+.d-flex {
+  display: flex;
+}
+
+.book-container {
+  font-family: 'Roboto Slab', serif;
+  justify-content: space-between;
+  align-items: center;
+  border: solid 1px rgb(214, 214, 214);
+  border-radius: 4px;
+  margin-left: -41px;
+  margin-top: 10px;
+  height: 7rem;
+  padding: 10px 5%;
+  background-color: #fff;
+}
+
+.general-info {
+  flex-flow: column wrap;
+  width: 30%;
+}
+
+.title {
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: #121212;
+  margin: 0;
+}
+
+.book-management {
+  list-style-type: none;
+  padding: 0;
+}
+
+.current-chapter > button {
+  width: 10rem;
+  height: 2.063rem;
+  margin-top: 1.6rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  background-color: #ff7402;
+  color: azure;
+  border: none;
+  border-radius: 3px;
+  font-size: 0.813rem;
+  letter-spacing: 0.5px;
+  cursor: pointer;
+}
+
+.book-management > li > button {
+  margin: 0 15px 0 0;
+  border: none;
+  cursor: pointer;
+  padding: 0;
+  font-family: 'Robotoslab', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.current-chapter > button:hover {
+  background-color: #036cbd;
+}
+
+.book-management > li > button:hover {
+  color: #004886;
+}
+
+.completion-detail {
+  align-items: center;
+}
+
+.progress-bar {
+  width: 80px;
+  height: 80px;
+  margin-right: 10px;
+}
+
+.completion-text {
+  flex-direction: column;
+}
+
+.completed-word {
+  opacity: 0.7;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  color: #121212;
+}
+
+.current-chapter {
+  flex-direction: column;
+}
+
+.genre {
+  color: #121212;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+}
+
+.author {
+  font-size: 0.875rem;
+  font-weight: 300;
+  letter-spacing: normal;
+  color: #ff7402;
+}
```

Comment:
### Your Project
![Student-BookStore - Copy](https://user-images.githubusercontent.com/93039359/205460983-fc6e5455-93a6-448c-a088-32c3d18693f8.PNG)

### Template
![bookstore](https://user-images.githubusercontent.com/93039359/205460992-c26e1e97-0bf0-45bf-acf4-50859cebe44f.png)

- [ ] Kindly use the correct color to make your project look similar to the given template 😊
- [ ] It would be nice if your project circle show the percentage as you can see in the template 👍